src/bio2bel/sources/rhea.py

```diff
@@ -18,7 +19,7 @@
 logger = logging.getLogger(__name__)
 
 MODULE_NAME = 'rhea'
-VERSION = '116'  # released on 2020-12-02
+VERSION = bioversions.get_version('rhea')
```

Comment:
since getting the version takes some amount of time, it's best to put inside the get_bel() function